src/List/columns/epics.js

```diff
@@ -50,44 +77,18 @@ const editColumnsForModel = (action$, store) =>
     action$
         .ofType(setColumnsTypes.request)
         .combineLatest(d2$)
-        .switchMap(async ([action, d2]) => {
-            let namespace;
-            try {
-                namespace = await d2.currentUser.dataStore.get(
-                    DATASTORE_NAMESPACE
-                );
-            } catch (e) {
-                if (e.httpStatusCode === 404) {
-                    //this is fine, we just need to create it
-                    try {
-                        namespace = await d2.currentUser.dataStore.create(
-                            DATASTORE_NAMESPACE
-                        );
-                    } catch (e) {
-                        //actual error
-                        logger.error('Failed to create new namespace!', e);
-                        return genericErrorMessage;
-                    }
-                } else {
-                    //Some other actual error
-                    logger.error(e);
-                    return genericErrorMessage;
-                }
-            }
-            try {
-                const cols = getAllModelTypes(store.getState());
-                const updatedColumns = setColumnsForModel(cols, action);
-                const ret = await namespace.set(COLUMN_KEY, updatedColumns);
-
-                return {
+        .switchMap(([action, d2]) =>
```

Comment:
This refactor was needed since closing the ColumnConfigDialog was done through the reducer for that dialog. As this PR also added centralized dialog-handling, and this dialog previousy had its own reducer (was nested in configurableColumns-reducer), I removed this.
Now this epic also dispatches ad action for closing once it completes.